### PR TITLE
Updated configuration.md for missing config properties

### LINKED ISSUE #204: enable_envvars_config appears to do nothing
AFAICT, the `enable_envvars_config` config file setting does nothing. The `LOG_LOCATION`, `METRICS_LOCATION`, and `TS_CONFIG_FILE` env vars are all picked up by TorchServe on startup. I tried this three ways in my `config.properties`:

* No `enable_envvars_config` entry
* `enable_envvars_config=true`
* `enable_envvars_config=false`

In all three trials, the env var behavior was as it should be for the case the docs describe for `enable_envvars_config=true`.

### PR DESCRIPTION
**Pull request by shivamshriwas**
## Description

Added missing config properties in configuration.md
Added an note on use of enable_envvars_config.
Modifed `MODEL_SERVER_HOME` env variable to `TS_MODEL_SERVER_HOME` to make it inline with other config properties which can be set using env vars.

Fixes #204 

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] This change requires a documentation update

## Feature/Issue validation/testing

- [X] Test A
set enable_envvars_config true in config.properties
Use  env variable to  set config property. (refer configuration.md)
Observe use of env variable in torchserve instance.

  
- UT/IT execution results

- Logs
- [ 
[issue_204_torchserve_sanity_log.txt](https://github.com/pytorch/serve/files/4616688/issue_204_torchserve_sanity_log.txt)

## Checklist:

] Have you added tests that prove your fix is effective or that this feature works?
- [x] New and existing unit tests pass locally with these changes?
- [x] Has code been commented, particularly in hard-to-understand areas?
- [x] Have you made corresponding changes to the documentation?

### CODE REVIEW

File: docs/configuration.md [hunk 1]
```diff
@@ -191,3 +191,18 @@ Most of the following properties are designed for performance tuning. Adjusting
 * `decode_input_request`: Configuration to let backend workers to decode requests, when the content type is known.
 If this is set to "true", backend workers do "Bytearray to JSON object" conversion when the content type is "application/json" and
 the backend workers convert "Bytearray to utf-8 string" when the Content-Type of the request is set to "text*". Default: true  
+* `debug`: runs Torchserve in debug mode where `default_workers_per_model` is set to 1 when this flag is true.
+* `model_store` : path of model store directory.
+* `model_server_home` : Torchserve home directory.
+* `max_request_size` : The maximum allowable request size that the Torchserve accepts.
```
[the remark this answers is not in the document]
Comment (shivamshriwas) [reply docs/configuration.md]: Done

File: docs/configuration.md [hunk 2]
```diff
@@ -191,3 +191,18 @@ Most of the following properties are designed for performance tuning. Adjusting
 * `decode_input_request`: Configuration to let backend workers to decode requests, when the content type is known.
 If this is set to "true", backend workers do "Bytearray to JSON object" conversion when the content type is "application/json" and
 the backend workers convert "Bytearray to utf-8 string" when the Content-Type of the request is set to "text*". Default: true  
+* `debug`: runs Torchserve in debug mode where `default_workers_per_model` is set to 1 when this flag is true.
```
[the remark this answers is not in the document]
Comment (shivamshriwas) [reply docs/configuration.md]: @maaquib Updated debug in config properties as suggested.
For detailed description on running to torchserve in debug mode we will be adding 
a separate doc file.

File: docs/configuration.md [hunk 3]
```diff
@@ -191,3 +191,18 @@ Most of the following properties are designed for performance tuning. Adjusting
...
+* `model_store` : path of model store directory.
```
[the remark this answers is not in the document]
Comment (shivamshriwas) [reply docs/configuration.md]: Done

**Comment by maaquib** [comment]
@shivamshriwas Can you also take a look at the min_worker scaling issue that I reported. If it is not related to your change, I'll create a separate issue. Thanks.

**Comment by shivamshriwas** [comment]
> @shivamshriwas Can you also take a look at the min_worker scaling issue that I reported. If it is not related to your change, I'll create a separate issue. Thanks.

@maaquib 
the issue here is not due to changes in this PR , its debug mode specific which is not intended to use as steps followed.
debug option is developer specific property hence as discussed removed from main doc.
issue #215 will be taking care of adding documentation around it.

File: docs/configuration.md [hunk 4]
```diff
@@ -191,3 +191,17 @@ Most of the following properties are designed for performance tuning. Adjusting
...
 the backend workers convert "Bytearray to utf-8 string" when the Content-Type of the request is set to "text*". Default: true  
+* `model_store` : Path of model store directory.
+* `model_server_home` : Torchserve home directory. 
+* `max_request_size` : The maximum allowable request size that the Torchserve accepts, in bytes. Default: 6553500
+* `max_response_size` : The maximum buffer size the frontend allocates for a worker response, in bytes. Default: 6553500
+
+---
+**NOTE**
+
+Config properties can be set using environment variable only when `enable_envvars_config` is true and
```
Comment (dhaniram-kshirsagar) [inline docs/configuration.md:202]: List down steps

Comment (shivamshriwas) [reply docs/configuration.md]: done

**Comment by datavistics** [comment]
Its called `max_request_size` in the config.properties file. 
Here is what Im using:
`max_request_size=999999999`

**Pull request merged**